src/api/public/apidocs-new/paths/request.yaml

```diff
@@ -0,0 +1,16 @@
+get:
+  summary: Get a simple directory listing of all requests
+  description: Get a simple directory listing of all requests
+  security:
+    - basic_authentication: []
+  responses:
+    '200':
+      description: OK
+      content:
+        application/xml:
+          schema:
+            $ref: '../components/schemas/request/directory.yaml'
```

Comment:
We could use the existing `../components/schemas/directory.yaml`.

Reply:
@eduardoj there is a slight difference between those schemas, the request one has the an additonal `count` attribute

Reply:
The `components/schemas/directory.yaml` schema doesn't require to define the `count` attribute. So that schema can still be used.

In this case, we could provide an example without the `count` attribute:

```yaml
       content:
        application/xml:
          schema:
            $ref: '../components/schemas/request/directory.yaml'
          example:                                                                                                                                       
            entry:                                                                                                                                       
              - name: Some text                                                                                                                          
```

Reply:
ahh okay, thanks